test/jdk/javax/swing/JOptionPane/bug8024926.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2013, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+import java.awt.BorderLayout;
+import java.awt.Dialog;
+import java.awt.EventQueue;
+import java.awt.Frame;
+import java.awt.TextArea;
+import javax.swing.JDialog;
+import javax.swing.JOptionPane;
+import javax.swing.SwingUtilities;
+
+/**
+ * @test
+ * @bug 8024926 8040279
+ * @requires (os.family == "mac")
+ * @summary [macosx] AquaIcon HiDPI support
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual bug8024926
+ */
+public class bug8024926 {
+
+    private static final String INSTRUCTIONS = """
+                Verify that high resolution system icons are used
+                 in JOptionPane on HiDPI displays.
+                1) Run the test on Retina display or enable the Quartz Debug
```

Comment:
Quartz debug ? How ? And we don't even use Quartz.
The pass criteria is completely subjective. 
So this test looks like it needs to be better.
If you can't figure out a better test for this in less than a day just delete this useless thing.

Reply:
I can remove the "Quartz debug" portion if it is not applicable, but as you know, it's a straight lift from the existing instructions...The reviewer that time did not object to this phrase it seems (was it used that time and subsequently revoked later?)
https://hg.openjdk.org/jdk9/jdk9/jdk/rev/4991a758abef

Also, the product fix has another test `test/javax/swing/JMenuItem/8031573/bug8031573.java`	which also put the same pass criteria that icons should be smooth and our manual testing team has not made any objections to the instructions of both test it seems..

Reply:
It looks like Quartz is still used in some native image handling code.
OK. If the instruction is harmless let it stay, but I doubt anyone does that.